packages/@aws-cdk/aws-servicecatalogappregistry/lib/application.ts

```diff
@@ -75,19 +141,26 @@ export class Application extends ApplicationBase {
 
   public readonly applicationArn: string;
   public readonly applicationId: string;
+  private readonly application: CfnApplication;
```

Comment:
No need for this field anymore.

Reply:
right, had added it back for the hashing but then by adding that as a field we can go back to how it originally was with just a `const`